lib/internal/blob.js

```diff
@@ -200,7 +199,7 @@ class Blob {
     const length = this[kLength];
     return {
       data: { handle, type, length },
-      deserializeInfo: 'internal/blob:ClonedBlob',
+      deserializeInfo: 'internal/blob:CloneableBlob',
```

Comment:
Since `CloneableBlob` extends the constructor of `Blob` and does nothing else, I believe we can go straight with:

```suggestion
      deserializeInfo: 'internal/blob:Blob',
```

Reply:
I think the `static` never worked, and I didn't saw the `markTransferMode` on the constructor, but makes sense this change, thanks for reviewing!